lib/config.js

```diff
@@ -1,13 +1,13 @@
 /*!
- * Copyright (c) 2020-2021 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
 import bedrock from 'bedrock';
 const {config} = bedrock;
 
-const namespace = 'module-template-http';
-const cfg = config[namespace] = {};
-
-const basePath = '/foo';
-cfg.routes = {
-  basePath
+config['oauth2-client'] = {
+  cache: {
```

Comment:
We may need some other caching in the future, so let's make this more specific:

```suggestion
  accessTokenCache: {
```